packages/cli/src/commands/dao_cmds/new.js

```diff
@@ -56,21 +54,19 @@ export const builder = yargs => {
 
 // Task will be moved to handler once `dao start` is refactored
 export const task = async ({
-  web3,
```

Comment:
Global options are no longer needed across all aragonCLI commands. Handle on the toolkit.